src/config/database.ts

```diff
@@ -12,7 +12,7 @@ const dbConfig: ConnectionOptions = {
   username: postgresUser,
   password: postgresPassword,
   database: postgresDatabase,
-  entities: [Ticker, Trade, Task],
+  entities: [Ticker, Trade, Task, TradeHistory],
```

Comment:
There is already entity `Trade` to log swap items.

```suggestion
  entities: [Ticker, Trade, Task],
```